### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by developerBarak**
Added AlertDialog and "Delete a Channel" Listtile to the Channel Info Screen

### CODE REVIEW

File: lib/ui/shared/dialogs/delete_channel_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    return StatefulBuilder(builder: (context, setState) {
+
+      return AlertDialog(
+        insetPadding:  EdgeInsets.symmetric(horizontal: 65.0, vertical: 24.0),
+        scrollable: true,
+        title: Text('Delete Channel'),
+        // titlePadding: EdgeInsets.fromLTRB(24.0, 24.0, 24.0, 0),
+        content: Text(
+          'Are you sure you want to delete this channel, this can’t be undone',
+          style: TextStyle(fontSize: 16.0),
```
Comment (Quwaysim) [inline lib/ui/shared/dialogs/delete_channel_dialog.dart:62]: Apply a style from AppTextStyles class

File: lib/ui/view/channel/channel_info/channel_info_view_model.dart [hunk 2]
```diff
@@ -7,10 +9,26 @@ import '../../../../app/app.router.dart';
 class ChannelInfoViewModel extends BaseViewModel {
   final _navigationService = locator<NavigationService>();
 
+  final _dialogService = locator<DialogService>();
+
+
+
+  Future showDialog() async{
+    print("Testing call");
```
Comment (Quwaysim) [inline lib/ui/view/channel/channel_info/channel_info_view_model.dart:17]: remove print statement, use logger if you want to print vars

File: lib/ui/view/channel/channel_info/widgets/seventh_section.dart [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                ),
+              ],
+            ),
+            Container(
+                alignment: Alignment.center,
+                margin: const EdgeInsets.only(left: 45, bottom: 10),
+                child: const Text(
+                  'All messages from this channel will be removed from Zuri Chat. Any files uploaded to this channel'
+                      'would not be removed.',
+                  style: TextStyle(),
```
Comment (Quwaysim) [inline lib/ui/view/channel/channel_info/widgets/seventh_section.dart:60]: Apply a style from AppTextStyles class

File: lib/ui/view/channel/channel_view/channel_page_viewmodel.dart [hunk 4]
```diff
@@ -6,11 +6,13 @@ import 'package:stacked_services/stacked_services.dart';
 class ChannelPageViewModel extends BaseViewModel {
   final _navigationService = locator<NavigationService>();
 
-  navigateToChannelInfo() {
-    _navigationService.navigateTo(Routes.channelInfoView);
+  Future navigateToChannelInfo() async{
+   await _navigationService.navigateTo(Routes.channelInfoView);
```
Comment (Quwaysim) [inline lib/ui/view/channel/channel_view/channel_page_viewmodel.dart:10]: why "awaiting"?

**Review by Quwaysim — CHANGES_REQUESTED** [review]
make changes

**Review by developerBarak — COMMENTED** [review]
Added AlertDialog and "Delete a Channel" Listtile to the Channel Info Screen and also made changes to the styles

File: lib/ui/view/channel/channel_view/channel_page_viewmodel.dart (hunk 4 above)
Comment (developerBarak) [reply lib/ui/view/channel/channel_view/channel_page_viewmodel.dart:10]: that's what i did to make my implementation work

File: lib/ui/view/channel/channel_info/channel_info_view_model.dart (hunk 2 above)
Comment (developerBarak) [reply lib/ui/view/channel/channel_info/channel_info_view_model.dart:17]: resolved.

File: lib/ui/view/channel/channel_info/widgets/seventh_section.dart (hunk 3 above)
Comment (developerBarak) [reply lib/ui/view/channel/channel_info/widgets/seventh_section.dart]: resolved

**Review by developerBarak — COMMENTED** [review]
resolved issues

**Review by Quwaysim — APPROVED** [review]
looks good now

**Review by mariamadebolahamzat — APPROVED** [review]
Changes made

**Pull request merged**